sdk/ai/azopenai/CHANGELOG.md

```diff
@@ -1,17 +1,15 @@
 # Release History
 
-## 0.5.2 (Unreleased)
+## 0.5.2 (2024-05-07)
```

Comment:
Looks like there were some breaking changes, so this should bump to `0.6.0`.

Reply:
I don't think there are any breaking changes (all my old tests passed without modification). 

If you're talking about the customError, that has to do with the test resources being oversubscribed, but it's not a contract change.